wordpress-mobile/WordPress-iOS
# Add stats for THAT segmented control and PostEpilogue

**Fixes** #8631

**To test:**

Do each one of the actions and test that they appear on Tracks:
1. Change notifications segmented control value.
2. Post Epilogue: Displayed, View, Edit and Share.

Thanks in advance
